Dotnet.Homeworks.Features/Products/Commands/InsertProduct/InsertProductCommand.cs

```diff
@@ -0,0 +1,11 @@
+namespace Dotnet.Homeworks.Features.Products.Commands.InsertProduct;
+
+public class InsertProductCommand
+{
+    public string Name { get; set; } = string.Empty;
```

Comment:
Здесь и ниже в других command/query классах без надобности не нужен set
```suggestion
    public string Name { get; init; }
```